packages/block-library/src/table/test/state.js

```diff
@@ -0,0 +1,251 @@
+/**
+ * Internal dependencies
+ */
+
+import {
+	createTable,
+	updateCellContent,
+	insertRow,
+	deleteRow,
+	insertColumn,
+	deleteColumn,
+} from '../state';
+
+const table = {
```

Comment:
Can we wrap `table` and `tableWithContent` with `deepFreeze` call to ensure there is no mutation?